Improve the Chinese translation of "Miscellaneous"

ifsheldon opened the pull request:
The previous translation of "Miscellaneous" to Chinese takes the meaning when the word is used as an adj. The new translation takes the meaning when the word is used as a none. "杂项", when back-translated to English, is "Miscellaneous items", which can be derived as "Miscellaneous settings" based on the context. It is a more suitable translation in my opinion.

BTW: Loving this app! THANKS a lot to every contributor.

lwouis commented:
@ifsheldon thank you a lot for contributing!

To change localization, please [go here](https://poeditor.com/join/project/8AOEZ0eAZE), and you will have a nice UI to update the text.

poeditor.com is our source-of-truth for localized strings, so I can't merge this PR. The change has to be done there in the editor, and then there are scripts which will bring the new values into the git project automatically.

An added benefit of updating through poeditor.com is that your name will be mentioned in the list of contributors, both on the website and in the app itself 👍

ifsheldon commented:
OK, I have changed the simplified Chinese translation of "Miscellaneous" and added traditional Chinese translation of it. I'm not familiar with poeditor.com, but it seems I have submitted the changes. Can you please check if everything is alright? Thanks!

lwouis commented:
It's correctly updated. It will ship in the app next release 👍 

Thank you~
